docs/Tutorials/Create-Privacy-enabled-network.md

```diff
@@ -17,335 +17,204 @@ to encrpyt and distribute [private transactions](../Concepts/Privacy/Privacy.md)
+the configuration below into the file.
+
+```json
+{
+   "useWhiteList": false,
+   "jdbc": {
+       "username": "sa",
+       "password": "",
+       "url": "jdbc:h2:/<path to IBFT-network>/IBFT-network/Tessera-0/db1;MODE=Oracle;TRACE_LEVEL_SYSTEM_OUT=0",
+       "autoCreateTables": true
+   },
+   "serverConfigs":[
+       {
+           "app":"ThirdParty",
+           "enabled": true,
+           "serverAddress": "http://localhost:9081",
+           "communicationType" : "REST"
        },
-       "serverConfigs":[
-           {
-               "app":"ThirdParty",
-               "enabled": true,
-               "serverAddress": "http://localhost:9081",
-               "communicationType" : "REST"
-           },
-           {
-               "app":"Q2T",
-               "enabled": true,
-                "serverAddress":"unix:/yourpath/new-node-1t/tm.ipc",
-               "communicationType" : "REST"
-           },
-           {
-               "app":"P2P",
-               "enabled": true,
-               "serverAddress":"http://localhost:9001",
-               "sslConfig": {
-                   "tls": "OFF"
-               },
-               "communicationType" : "REST"
-           }
-       ],
-       "peer": [
-           {
-               "url": "http://localhost:9001"
+       {
+           "app":"Q2T",
+           "enabled": true,
+           "serverAddress":"unix:/<path to IBFT-network>/IBFT-network/Tessera-0/tm.ipc",
+           "communicationType" : "REST"
+       },
+       {
+           "app":"P2P",
+           "enabled": true,
+           "serverAddress":"http://localhost:9001",
+           "sslConfig": {
+               "tls": "OFF"
            },
+           "communicationType" : "REST"
+       }
+   ],
+   "peer": [
+       {
+           "url": "http://localhost:9001"
+       },
+       {
+           "url": "http://localhost:9003"
+       }
+   ],
+   "keys": {
+       "passwords": [],
+       "keyData": [
            {
-               "url": "http://localhost:9003"
+               "privateKeyPath": "<path to IBFT-network>/IBFT-network/Tessera-0/tessera0.key",
+               "publicKeyPath": "<path to IBFT-network>/IBFT-network/Tessera-0/tessera0.pub"
            }
-       ],
-       "keys": {
-           "passwords": [],
-           "keyData": [
-               {
-                   "privateKeyPath": "/yourpath/new-node-1t/new-node-1.key",
-                   "publicKeyPath": "/yourpath/new-node-1t/new-node-1.pub"
-               }
-           ]
+       ]
+   },
+   "alwaysSendTo": []
+}
+```
+
+### 4. Create Tessera-1 keys
+
+In `Tessera-1`, generate keys in the same way as for `Tessera-0`.
+
+```bash
+java -jar /<path-to-tessera>/tessera.jar -keygen -filename tessera1
+```
+
+### 5. Create Tessera-1 configuration file
+
+In the `Tessera-1` directory, create a configuration file called `config.json`. Copy and paste the
+the configuration below into the file. Different ports are specified for Tessera 1.
```

Comment:
Ask for updating `<path to IBFT-network>` with real path here too.

Reply:
Fixed